test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -675,8 +674,7 @@ def test_cgroup_cpuset_and_memory(self):
             with open(tmp_file, 'r') as fd:
                 tmp_out = fd.read().splitlines()
         except:
-            self.logger.info("Job did not produce expected output")
-            self.assertTrue(False)
+            self.assertTrue(False, "Job did not produce expected output")
```

Comment:
Don't do assertTrue(False).  Use self.fail(msg) (same for the rest of the test plan)